pyproject.toml

```
@@ -65,7 +65,7 @@ trio = "^0.19"
 attrs = "^21.2"
 httpx = "^0.21"
 
-mypy = "^0.910"
+mypy = "0.920"
```

```suggestion
mypy = "^0.920"
```

We don't pin `mypy`, this is a dev-only dep.